### PR TITLE
IA-2972: Double check the mobile feature flags descriptions

### PR DESCRIPTION
**Pull request by hakifran**
Explain what problem this PR is resolving
- Double check the mobile feature flags descriptions
- Have mobile feature flags available in French
Related JIRA tickets : [IA-2972](https://bluesquare.atlassian.net/browse/IA-2972)
## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [x] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes
- Add french translations for featureFlags
- Fix tooltip description

## How to test

- Go in projects
- Change the language to french
- Check if featureFlags descriptions are correct

[IA-2972]: https://bluesquare.atlassian.net/browse/IA-2972?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json [hunk 1]
```diff
@@ -1014,46 +1014,55 @@
...
+    "iaso.projets.featureflag.mobile_check_forms_update": "Mobile: Warn the user when forms have been updated",
+    "iaso.projets.featureflag.mobile_check_ou_update": "Mobile: Warn the user when the org units have been updated",
+    "iaso.projets.featureflag.mobile_entity_limited_search": "Mobile: Limit entities search",
     "iaso.projets.featureflag.mobile_entity_no_creation": "Mobile: User cannot create a entity",
-    "iaso.projets.featureflag.mobile_entity_warn_when_found": "Mobile: Display a message when an entity is found in the duplicate search",
-    "iaso.projets.featureflag.mobile_finalized_form_are_read": "Mobile: Finalized forms are read only",
-    "iaso.projets.featureflag.mobile_force_forms_update": "Mobile: Warn the user when forms have been updated and force them to update.",
-    "iaso.projets.featureflag.mobile_force_ou_update": "Mobile: Warn the user when the Org Units have been updated and force them to update.",
+    "iaso.projets.featureflag.mobile_entity_warn_when_found": "Mobile: display a message when an entity is found on the duplicate search",
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/app/translations/en.json]: Display

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json [hunk 2]
```diff
@@ -1014,46 +1014,55 @@
...
-    "iaso.projets.featureflag.mobile_submission_incomplete_by_default": "Do not check 'Finalized' by default at the end of ODK forms",
-    "iaso.projets.featureflag.org_unit_change_request": "Request changes to org units.",
+    "iaso.projets.featureflag.mobile_org_unit_registry": "Mobile: Change requests",
+    "iaso.projets.featureflag.mobile_submission_incomplete_by_default": "Do not check 'finalized' by default at the end of the form",
+    "iaso.projets.featureflag.org_unit_change_request": "Request changes to org units",
     "iaso.projets.featureflag.planning": "Mobile: Show planning screen",
     "iaso.projets.featureflag.reports": "Enable reports",
     "iaso.projets.featureflag.require_authentication": "Authentication",
-    "iaso.projets.featureflag.show_detail_map_on_mobile": "Mobile: Show map of OrgUnit",
+    "iaso.projets.featureflag.show_detail_map_on_mobile": "Mobile: Show Map of org unit",
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/app/translations/en.json:1039]: map

**Review by beygorghor — DISMISSED** [review]
Some capitalising issues

File: hat/assets/js/apps/Iaso/domains/projects/messages.ts [hunk 3]
```diff
@@ -96,76 +101,85 @@ const MESSAGES = defineMessages({
...
+    mobile_entity_warn_when_found_tooltip: {
+        id: 'iaso.projets.featureflag.tooltip.mobile_entity_warn_when_found',
+        defaultMessage:
+            'A message prompts when an entity duplicate has been found',
     },
     check_position_for_forms: {
         id: 'iaso.projets.featureflag.check_position_for_forms',
         defaultMessage:
-            'Mobile: Enforce users are within reach of the Org Unit before starting a form.',
+            'Mobile: enforce users are within reach of the org unit before starting the form',
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/projects/messages.ts]: Enforce

File: hat/assets/js/apps/Iaso/domains/projects/messages.ts [hunk 4]
```diff
@@ -96,76 +101,85 @@ const MESSAGES = defineMessages({
...
         id: 'iaso.projets.featureflag.tooltip.mobile_finalized_form_are_read',
         defaultMessage:
-            'Tab in the mobile application to show the geographic information available for the selected Org Unit',
+            'Blocks the edition of forms from the mobile app once finalized',
     },
     limit_ou_download_to_roots: {
         id: 'iaso.projets.featureflag.limit_ou_download_to_roots',
         defaultMessage:
-            'Mobile: Limit download of orgunit to what the user has access to',
+            'Mobile: limit download of org unit to what the user has access to',
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/projects/messages.ts]: Limit

**Review by quang-le — DISMISSED** [review]
Flagged some missing uppercase in default messages. pls check the json file as well

**Review by mathvdh — APPROVED** [review]

**Pull request merged**